src/applications/gi/containers/SearchPage.jsx

```diff
@@ -230,14 +231,53 @@ export class SearchPage extends React.Component {
     );
 
     const searchResults = this.searchResults();
+
     return (
       <ScrollElement name="searchPage" className="search-page">
+        {!environment.isProduction() &&
```

Comment:
why is this here twice?
if it needs to be twice extract out into a renderLogo function